src/Microsoft.OData.Client/Descriptor.cs

```diff
@@ -125,6 +135,27 @@ internal virtual bool IsModified
             }
         }
 
+        /// <summary>DependsOnIds for the modified entity.</summary>

```

Comment:
@KenitoInc How about using auto-properties?

Reply:
I had thought of the same, however other properties in the same file don't used autoproperties.
Maybe we can cleanup this code later. What do you think?